app/services/local-data.js

```diff
@@ -0,0 +1,39 @@
+import Service from '@ember/service';
+import { isEmpty } from '@ember/utils';
+import config from 'hyperchannel/config/environment';
+import * as localforage from 'localforage';
+
+const defaultValues = {
+  userSettings: {
+    nickname: null,
+    currentSpace: config.defaultSpaceId,
+    currentChannel: 'kosmos'
+  }
+};
+
+export default class LocalDataService extends Service {
+
+  constructor () {
+    super(...arguments);
+    window.localforage = localforage;
+    this.stores = {
+      userSettings: localforage.createInstance({
+        name: 'hyperchannel', storeName: 'userSettings'
+      })
+    }
+  }
+
+  async setDefaultValues () {
+    for (const storeName of Object.keys(defaultValues)) {
```

Comment:
What you can try is assign `const defaultValuesKeys = Object.keys(defaultValues);` outside the class...

Reply:
To make CodeClimate happy

Reply:
Didn't help. :/